nextcord/types/polls.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: MIT
+
+from typing import List, Literal, Optional, TypedDict
+
+from typing_extensions import NotRequired
+
+from .emoji import PartialEmoji
+
+LayoutType = Literal[1]
+
+
+class PollMedia(TypedDict):
+    text: str
+    emoji: NotRequired[PartialEmoji]
+
+
+class PollAnswer(TypedDict):
+    poll_media: PollMedia
+
+
+class DAPIPollAnswer(TypedDict):
+    answer_id: int
+    poll_media: PollMedia
```

Comment:
I'd personally have this inherit as they are both on the same table in the discord docs, but just a nitpick.